.gitignore

```diff
@@ -31,6 +31,7 @@ migrate_working_dir/
 .flutter-plugins-dependencies
 .pub-cache/
 .pub/
+.vscode/
```

Comment:
No need to ignore .vscode.
Better to include it so all developers can use same configuration.